src/go/k8s/pkg/resources/statefulset_scale.go

```diff
@@ -109,10 +111,12 @@ func (r *StatefulSetResource) handleScaling(ctx context.Context) error {
 	}
 
 	// User required replicas is lower than current replicas (currentReplicas): start the decommissioning process
+	r.logger.Info("Downscaling cluster", "replicas", *r.pandaCluster.Spec.Replicas)
```

Comment:
NIT: Rise Kubernetes Event for better tracking?

Reply:
The whole controller needs proper event tracking. I'm leaving that out of scope for this PR.